pegasus/sites.v3/code.org/public/css/dance-landing/dance.css

```diff
@@ -142,10 +146,10 @@ a:hover {
   }
 
   .img-container {
-    background: url(/images/dance-hoc/dance_party_tutorial_hero.png);
     margin: 5px;
     border-radius: 5px;
     background-size: cover;
+    padding: initial;
```

Comment:
...and the image on top was not entirely covering that image because it had 5px of padding on the left & right.